cmds/fmap/lib/fmap_test.go

```diff
@@ -6,6 +6,8 @@ package fmap
 
```

Comment:
pretty awesome that each time you write a PR you write the test. We've been pretty sloppy about this until now.